### ISSUE TITLE
(Option to) ensure that pinch-zoom functions on Android

### ISSUE THREAD

**Issue by dkrasne** [observablehq/framework#2001]
Observable Framework appears not to allow pinch-zooming on a mobile device, by default. This is now overriden in iOS, which I believe most of the developers use, but not in Android. For accessibility purposes, it should be possible to have a setting (presumably in the header material or configuration) that stops forcing viewport size. (If this is already supported, then the suggested enhancement would instead be to have it clearly indicated in the documentation...)

Thanks!
